explore/heatmap-vega-lite/heatmap.py

```diff
@@ -123,14 +124,14 @@
 y = y_full.assign(indicator = y_full.max(axis=1))
 
 
-# In[12]:
+# In[11]:
 
 unique_pos = y.groupby('disease').apply(lambda x: x['indicator'].sum())
 heatmap_df0 = y_full.groupby('disease').sum().assign(TOTAL = unique_pos)
```

Comment:
Can we switch `TOTAL` to `Total`? Gene names are CAPS by convention, but total is a regular word. (We don't want someone thinking total is a gene)